spring/src/main/java/org/axonframework/spring/config/AnnotationDriven.java

```diff
@@ -20,20 +20,31 @@
 import org.axonframework.queryhandling.QueryHandler;
 import org.springframework.context.annotation.Import;
 
-import java.lang.annotation.*;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
 /**
  * Annotation for {@link org.springframework.context.annotation.Configuration @Configuration} that will automatically
  * subscribe {@link CommandHandler @CommandHandler} and {@link QueryHandler @QueryHandler} annotated beans with the
  * CommandBus and QueryBus, respectively.
  *
  * @author Allard Buijze
+ * @see MessageHandlerLookup
+ * @see SpringSagaLookup
+ * @see SpringAggregateLookup
+ * @see SpringConfigurer
+ * @see SpringAxonConfiguration
  * @since 2.3
+ * @deprecated Use Spring Boot Autoconfiguration or register the individual beans explicitly. See the list of
```

Comment:
I think we're missing something in this deprecated description:

```suggestion
 * @deprecated Use Spring Boot Autoconfiguration or register the individual beans explicitly. Check the "See also" list for which individual beans to register.
```